lunavl/sdk/faceengine/setting_provider.py

```diff
@@ -209,72 +209,45 @@ def setValue(self, name: str, value: Any) -> None:
         """
         self._coreSettingProvider.setValue(self.__class__.sectionName, name, CoreFE.SettingsProviderValue(value))
 
-    def getValue(self, name: str) -> Any:
+    def getValue(self, name: str) -> Optional[Any]:
         """
         Get setting value
         Args:
             name: setting name
 
         Returns:
-            a value
+            a value or None if settings does not exists
         """
-        return self._coreSettingProvider.getValue(self.__class__.sectionName, name)
-
-    def getValueAsString(self, name: str) -> str:
-        """
-        Get setting value as string
-        Args:
-            name: setting name
-
-        Returns:
-            a string
-        """
-        return self.getValue(name).asString()
-
-    def getValueAsInt(self, name: str) -> int:
-        """
-        Get setting value as int
-        Args:
-            name: setting name
-
-        Returns:
-            a int
-        """
-        return self.getValue(name).asInt()
-
-    def getValueAsFloat(self, name: str) -> float:
-        """
-        Get setting value as float
-        Args:
-            name: setting name
-
-        Returns:
-            a float
-        """
-        return self.getValue(name).asFloat()
+        value = self._coreSettingProvider.getValue(self.__class__.sectionName, name)
+        if value is []:
+            return None
+        return value[0]
```

Comment:
```suggestion
        value = self._coreSettingProvider.getValue(self.__class__.sectionName, name)
        if not value:
            return None
        return value[0]
```
![image](https://user-images.githubusercontent.com/17934618/106277762-a7f6bf80-624a-11eb-8cef-9f819d29a925.png)

Reply:
fixed